src/cloud-element-templates/linting/LinterPlugin.js

```diff
@@ -37,12 +37,28 @@ export const elementTemplateLintRule = ({ templates = [] }) => {
 
   elementTemplates.set(validTemplates);
 
+  function setEngines(engines) {
+    if (engines == {}) {
```

Comment:
`==` does not work on `{}`. `{} == {}` will always be false (different identity).

Reply:
Thanks. Let's wait with review until I push "full" linting after latest discussions.

Reply:
BTW I'd expect the linter to enforce `===`